lib/main.dart

```diff
@@ -83,11 +75,7 @@ class HomeScreen extends StatelessWidget {
               FlutterConfig.get('SECRET'),
               style: const TextStyle(color: Colors.black, fontSize: 24),
             ),
-            const SizedBox(height: 24),
-            ElevatedButton(
-              onPressed: () => context.go('/$routePathSecondScreen'),
-              child: const Text("Navigate to Second Screen"),
-            ),
+            const SizedBox(height: 24)
```

Comment:
Please remove the unused `HomeScreen` and `SecondScreen` classes. We might need to remove the related test files as well.